db/migrate/20200421154306_alter_admins.rb

```diff
@@ -0,0 +1,13 @@
+class AlterAdmins < ActiveRecord::Migration[6.0]
+  def up
+    add_column("admins", "username", :string, :limit => 25, :null => false, :after => "email")
```

Comment:
remove parentheses. Just like you did on the next two lines - CONSISTENCY IS KING!
Also, we usually use symbols instead of strings for the table and column names, e.g. `"admins"` -> `:admins`